Sources/MongoSwift/APM.swift

```diff
@@ -529,56 +525,42 @@ public enum MongoEventType {
 }
```

Comment:
The above enum can be removed now, right?

Reply:
good point, removed